yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
why is this added? 

same for yarn integrity

Reply:
Good catch! That was a mistake, I tried to run yarn, yarn build and then remembered this isn't a docusaurus repo :)

Reply:
Removed both in the latest commit